README.md

```diff
@@ -189,9 +177,9 @@ $ make dev
 
 Kong relies on three test suites using the [busted] testing library:
 
-* Unit tests
-* Integration tests, which require Postgres and Cassandra to be up and running
-* Plugins tests, which require Postgres to be running
+- Unit tests
+- Integration tests, which require Postgres and Cassandra to be up and running
+- Plugins tests, which require Postgres to be running
```

Comment:
AFAICT, most markup engines treat '*' and '-' as equivalent when defining lists.